### PR TITLE
Add debug module to package

### PR DESCRIPTION
**Pull request by smooresni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nitsm-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adds a debug module to the package. Allows the following:
```python
import nitsm.debug

nitsm.debug.prompt_attach_debugger()
```

### Why should this Pull Request be merged?

We should make it easier for our end users to debug their code.  By packaging this with `nitsm` we're enabling consumers a streamlined path to debugging their code modules.

### What testing has been done?

Since a message box is shown, this module requires manual testing.  It will not be covered in our automated tests.

- [x] I have run the automated tests (required if there are code changes)

### CODE REVIEW

File: src/nitsm/debug.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    Pauses the Python interpreter and displays the process ID (PID). The PID can be used by an IDE
+    such as PyCharm to attach to the process for debugging. This is useful for stepping into nitsm
+    code modules from TestStand.
+
+    Instructions for use with PyCharm:
+        1. Call this function from the code module you want to debug. Placing it at the beginning
+            of the code module is recommended.
+        2. Add a breakpoint at the location where you want to start debugging. Make sure this
+            breakpoint will be reached after this function is called.
+        3. Execute a sequence that calls into the code module from TestStand.
```
Comment (ni-erikcrank) [inline src/nitsm/debug.py:16]:
```suggestion
        3. In TestStand, execute a sequence that calls into the code module.
```

**Review by ni-erikcrank — DISMISSED** [review]

File: src/nitsm/debug.py (hunk 1 above)
Comment (smooresni) [reply src/nitsm/debug.py]: https://github.com/ni/nitsm-python/pull/131/commits/34c0e33efecc1416566578f5dc329dcd447ab582

Comment (smooresni) [reply src/nitsm/debug.py]: https://github.com/ni/nitsm-python/pull/131/commits/18c70b4c4635af9a63a75c0638414a69fcc6b316

**Pull request merged**